### PR TITLE
Add Pair Recording application use case

### PR DESCRIPTION
**Pull request by pgetta**

### CODE REVIEW

File: source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+necessary circuit information, e.g. circuit info, cell locations and
+properties, so this might take some time) an interactive circuit visualisation
+will appear with controls facilitating cell selection.
+
+.. image:: images/step_2.png
+   :width: 800 px
+
+Cells in the view are colour-coded by one of their's properties, user can
+change this property with a `Color by` select on the bottom of the view.
+Form to the right consists of various controls and allows user to filter and
```
Comment (alex4200) [inline source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst]: **The** form to the right ...

File: source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Cells can be selected for later simulations by clicking on them in the
+interactive view or, if known beforehand, specifying their `gid`s in the form
+and click the `Add` button.
+
+.. image:: images/step_3.png
+   :width: 800 px
+
+When cell selection is done a user can proceed to simulation configuration by
+clicking a corresponding button which will load and render morphologies for
```
Comment (alex4200) [inline source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst]: ... morphologies for **the** selected cells.

File: source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+selected cells.
+
+.. image:: images/step_4.png
+   :width: 800 px
+
+A user can add stimulation and/or recordings by clicking on sections of
+interest and choosing a type of instrument being added (stimulus, recording or
+synaptic input - which is available only if clicking on soma). Each added
```
Comment (alex4200) [inline source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst]: ... or synaptic input - the latter is only available for a soma).

File: source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst (hunk 1 above)
Comment (pgetta) [reply source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst]: Done

File: source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst (hunk 2 above)
Comment (pgetta) [reply source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst]: Done

File: source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst (hunk 3 above)
Comment (pgetta) [reply source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst]: Changed as well

**Comment by lbologna** [comment]
Hello,
@alex4200 @pgetta I did some minor changes and put the page to DEV:
https://lbologna.github.io/hbp-sp6-guidebook/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.html

Please let me know if it's ok with you before moving to PROD.
Thank you.

**Comment by alex4200** [comment]
Hi Luca,


to me it looks good. Ok to be moved to production from my side.



Cheers

Alex

________________________________

**Comment by pgetta** [comment]
Thank you, @lbologna, looks good to me as well.

**Comment by lbologna** [comment]
Ok, thanks. Pushed to Prod.

**Pull request merged**